docs/mkdocs/docs/features/trailing_commas.md

```diff
@@ -0,0 +1,88 @@
+# Trailing Commas
+
+Like comments, this library does not support trailing commas in arrays and objects *by default*.
+
+You can pass set parameter `allow_trailing_commas` to `#!cpp true` in the parse function to allow trailing commas in arrays and objects. Trailing commas are ignored during parsing. Note that a single comma as the only content of the array or object (`[,]` or `{,}`) is not allowed.
+
+This library does not add trailing commas when serializing JSON data.
+
+!!! example
+
+    Consider the following JSON with trailing commas.
+
+    ```json
+    {
+        "planets": [
+            "Mercury",
+            "Venus",
+            "Earth",
+            "Mars",
+            "Jupiter",
+            "Uranus",
+            "Neptune",
+        ]
+    }
+    ```
+    
+    When calling `parse` without additional argument, a parse error exception is thrown. If `allow_trailing_commas` is set to `#! true`, the trailing commas are ignored during parsing:
+
+    ```cpp
+    #include <iostream>
+    #include "json.hpp"
+    
+    using json = nlohmann::json;
+    
+    int main()
+    {
+        std::string s = R"(
+        {
+            "planets": [
+                "Mercury",
+                "Venus",
+                "Earth",
+                "Mars",
+                "Jupiter",
+                "Uranus",
+                "Neptune",
+            ]
+        }
+        )";
+        
+        try
+        {
+            json j = json::parse(s);
+        }
+        catch (json::exception &e)
+        {
+            std::cout << e.what() << std::endl;
+        }
+        
+        json j = json::parse(s,
+                             /* callback */ nullptr,
+                             /* allow exceptions */ true,
+                             /* ignore_comments */ false,
+                             /* allow_trailing_commas */ true);
+        std::cout << j.dump(2) << '\n';
+    }
+    ```
```

Comment:
Move this code into an example file (docs/mkdocs/docs/example). The documentation generation will then compile it and create the output. You can then include both in the documentation via

```markdown
--8<-- "examples/meta.output"
```

(see other documentation files for examples)

This has the advantage of making sure the example remains compilable in the future.